Makefile

```diff
@@ -127,26 +127,26 @@ clean:
 .PHONY: ko
 ko:
 	LDFLAGS="$(LDFLAGS)" GIT_HASH=$(GIT_HASH) GIT_VERSION=$(GIT_VERSION) \
-	ko publish --base-import-paths --bare \
+	ko publish --base-import-paths --bare --sbom=spdx \
```

Comment:
This is the default so you don't need it. Removing it will keep `make ko` working for developers who aren't running `ko` from HEAD.

Reply:
makes sense, thank you 🤝

Reply:
Just revert all of `Makefile` -- if the `ko` version is from HEAD, it will automatically upload SBOMs in SPDX format. If it's from a release, it won't do anything different.

Reply:
ah, okay.